tests/read_pr.py

```diff
@@ -0,0 +1,175 @@
+import glob
+
+
+class ReadPR:
+    """
+    Recursive find '*** Keywords ***' text relationship for the files that were changed.
+    For each match retrieve and report what are the tags that we should run the test suite.
+    """
+
+    def __init__(self):
+        self.keyword_key = '*** Keywords ***'
+        self.magic_tag_separator = '_______'
+        self.magic_tag = '[Tags]'
+
+    def run(self):
+        data = self.get_diff()
+        files_changed = []
+        for line in data:
+            if line.startswith('diff --git '):
+                files_changed.append(line)
```

Comment:
list comprehension could be good solution as well. 

` files_changed = [line for line in data if line.startswith('diff --git ')]`

Reply:
Done